app/components/lg-lion-search.js

```diff
@@ -40,7 +46,8 @@ export default Ember.Component.extend({
         selectedGender = this.get('selectedGender'),
         defaultGender = this.get('defaultGender'),
         selectedOrganization = this.get('selectedOrganization'),
```

Comment:
if you are only using the selected organization's id in this method (and I think you are), you can do
`selectedOrganizationId = this.get('selectedOrganization.id')`. It will be null if there's no selected organization.